huxley/core/middlewares.py

```diff
@@ -1,24 +1,34 @@
 # Copyright (c) 2011-2015 Berkeley Model United Nations. All rights reserved.
 # Use of this source code is governed by a BSD License (see LICENSE).
 
-import json, logging
+import json, logging, sys, traceback
+
+from huxley.settings.main import DEBUG
 
 class ExceptionLoggerMiddleware(object):
-    def process_exception(self, request, exception):
-        logger = logging.getLogger('huxley')
-        logger.exception(exception)
 
-class ServerLoggingMiddleware(object):
     def process_exception(self, request, exception):
-        logger = logging.getLogger('huxley.server')
-        logger.exception(exception)
+        logger = logging.getLogger('huxley.exception')
+        exc_type, exc_value, exc_traceback = sys.exc_info()
+        exc_traceback = ''.join(traceback.format_exception(exc_type, exc_value, exc_traceback))
+        log = json.dumps({
+              'message': exc_traceback,
+              'uri': request.path,
+              'status_code': 500,
+              'username': request.user.username})
+        logger.error(log)
```

Comment:
I believe so. `logger.error` eventually calls its `emit` method which we have overwritten in the DatabaseHandler to expect a json string that it parses in order to create a LogEntry object. I don't think `logger.exception` calls `emit` but instead gets the exceptions traceback and prints it to the console.